docs/book/Dockerfile

```diff
@@ -1,17 +0,0 @@
-# Copyright 2018 The Kubernetes Authors.
```

Comment:
It might be nice to preserve a Dockerfile for local testing/development.

Reply:
This shouldn’t have been removed, nice catch!